app/controllers/help_controller.rb

```diff
@@ -1,4 +1,8 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    gon.preloads[:chat_enabled] = AppConfig.chat.enabled ? 'enabled' : 'disabled'
```

Comment:
Also passing `true`/`false` to to it as returned by `AppConfig.chat.enabled?` should be just fine.